draft-ietf-tls-tls13.md

```diff
@@ -1173,6 +1173,7 @@ numerical information may be omitted.
 
        enum { low, medium, high } Amount;
 
+For numerical values, the notation \<floor..ceiling\> is also allowed.
```

Comment:
For enums we use parens rather than angle brackets, so this would be "(floor..ceiling)". (we've generally used angle brackets for size ranges and parens for value ranges) With this tweak, I think this is a logical clarification to add.